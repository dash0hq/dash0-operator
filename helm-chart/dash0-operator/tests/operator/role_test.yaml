suite: test role
templates:
  - operator/role.yaml
tests:
  - it: leader election role should match snapshot
    asserts:
      - matchSnapshot: {}