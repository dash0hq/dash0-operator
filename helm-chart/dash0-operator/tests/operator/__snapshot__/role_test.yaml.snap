leader election role should match snapshot:
  1: |
    apiVersion: rbac.authorization.k8s.io/v1
    kind: Role
    metadata:
      labels:
        app.kubernetes.io/component: rbac
        app.kubernetes.io/instance: leader-election-role
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: role
        app.kubernetes.io/part-of: dash0-operator
        app.kubernetes.io/version: 1.0.0
        helm.sh/chart: dash0-operator-1.0.0
      name: dash0-operator-leader-election-role
      namespace: NAMESPACE
    rules:
      - apiGroups:
          - ""
        resources:
          - configmaps
        verbs:
          - get
          - list
          - watch
          - create
          - update
          - patch
          - delete
      - apiGroups:
          - coordination.k8s.io
        resources:
          - leases
        verbs:
          - get
          - list
          - watch
          - create
          - update
          - patch
          - delete
      - apiGroups:
          - ""
        resources:
          - events
        verbs:
          - create
          - patch
